tangelo/tangelo/__main__.py

```diff
@@ -168,6 +172,10 @@ def main():
     p.add_argument("-np", "--no-drop-privileges", action="store_const", const=True, default=None, help="disable privilege drop when started as superuser")
     p.add_argument("-s", "--sessions", action="store_const", const=True, default=None, help="enable session tracking (default)")
     p.add_argument("-ns", "--no-sessions", action="store_const", const=True, default=None, help="disable session tracking")
+    p.add_argument("--list-dir", action="store_const", const=True, default=None, help="enable directory content serving")
+    p.add_argument("--no-list-dir", action="store_const", const=True, default=None, help="disable directory content serving")
+    p.add_argument("--show-py", action="store_const", const=True, default=None, help="enable Python service source code serving")
+    p.add_argument("--no-show-py", action="store_const", const=True, default=None, help="disable Python service source code serving")
```

Comment:
`--no-list-dir` and `--no-show-py` should probably explicitly say "(default)" in their documentation so the user knows the default